package.json

```diff
@@ -37,34 +37,45 @@
   },
   "license": "MIT",
   "dependencies": {
-    "async": "1.5.1",
-    "commander": "2.9.0",
-    "express": "4.13.4",
-    "mailparser": "0.6.2",
-    "open": "0.0.5",
-    "rimraf": "^2.6.1",
-    "smtp-connection": "2.3.1",
-    "smtp-server": "1.16.1",
-    "socket.io": "1.7.3",
-    "wildstring": "1.0.8"
+    "async": "2.6.2",
+    "commander": "2.19.0",
+    "express": "4.16.4",
+    "mailparser-mit": "^1.0.0",
+    "opn": "^5.5.0",
+    "rimraf": "^2.6.3",
+    "smtp-connection": "4.0.2",
+    "smtp-server": "3.5.0",
+    "socket.io": "2.2.0",
+    "wildstring": "1.0.9"
```

Comment:
Wow, lots of great updates here! @oktapodia Are there any breaking changes w/ major version increases in these libraries, ex. smtp-connection or smtp-server?

Reply:
I only updated the packages which are not including a major breaking change.

The main changes are:
- Library `opn` instead of `open` (deprecated + contains a vulnerability)
- Library `mailparser-mit` instead of `mailparser` (the latest version is slightly different, the mit version is the same one) 

Another PR will come to update the other packages + some rewrite + optimized tests